geopy/geocoders/__init__.py

```diff
@@ -105,6 +105,7 @@
 from geopy.geocoders.opencage import OpenCage
 from geopy.geocoders.openmapquest import OpenMapQuest
 from geopy.geocoders.osm import Nominatim
+from geopy.geocoders.pickpoint import PickPoint
```

Comment:
I'm going to make a checklist for adding new geocoder classes next days (I'll attach the link when it's done), but before it's there, here's a couple of things definitely missing here:

1. `PickPoint` should be listed in the `__all__`  tuple [above in this file](https://github.com/geopy/geopy/blob/40633f53ce4b85e8e6a7cb0d62d9f0b35864050f/geopy/geocoders/__init__.py#L74)
2. Same for `SERVICE_TO_GEOCODER` dict ([below in this file](https://github.com/geopy/geopy/blob/40633f53ce4b85e8e6a7cb0d62d9f0b35864050f/geopy/geocoders/__init__.py#L125))
3. Should have own test module (done) which should be referenced in [`geopy/test/geocoders/__init__.py`](https://github.com/geopy/geopy/blob/40633f53ce4b85e8e6a7cb0d62d9f0b35864050f/test/geocoders/__init__.py) (missing)
4. In case an API KEY is required, it should be listed in [`geopy/test/geocoders/util.py` (see the top of the file)](https://github.com/geopy/geopy/blob/40633f53ce4b85e8e6a7cb0d62d9f0b35864050f/test/geocoders/util.py#L15) and the test class should be guarded with a unittest decorator (see [`test/geocoders/what3words.py`](https://github.com/geopy/geopy/blob/40633f53ce4b85e8e6a7cb0d62d9f0b35864050f/test/geocoders/what3words.py#L18) for example).